src/api/entities/UserRole.ts

```diff
@@ -0,0 +1,11 @@
+import { BaseModel } from './BaseModel';
```

Comment:
This model doesn't need any relationMappings because the mappings are in the UserToParticipantRole mapping entity